packages/fether-react/src/Send/TxForm/TxForm.js

```diff
@@ -209,14 +209,20 @@ class Send extends Component {
         return preValidation;
       }
 
+      // If the gas hasn't been calculated yet, then we don't show any errors,
+      // just wait a bit more
+      if (!values.gas) {
+        return;
+      }
+
       if (!ethBalance || isNaN(values.gas)) {
         throw new Error('No "ethBalance" or "gas" value.');
       }
 
       // Verify that `gas + (eth amount if sending eth) <= ethBalance`
       if (
         values.gas
-          .mul(toWei(values.gasPrice, 'shannon'))
+          .multipliedBy(toWei(values.gasPrice, 'shannon'))
           .plus(token.address === 'ETH' ? toWei(values.amount) : 0)
```

Comment:
`token.address`(i.e. `0x005072Fb985cc64d8A1Ce00ECe26279f73c12F8a`) should be `token.symbol` (i.e. `ETH`)

Reply:
No, token.address is an unique field, and we consider in Fether Ethereum as a token, to factorize some code.

Reply:
when i'm in "Send Ether" and debug the value of `token` it displays:

<img width="211" alt="screen shot 2018-12-18 at 7 35 48 pm" src="https://user-images.githubusercontent.com/6226175/50168405-6c176f00-02fc-11e9-8812-a5cfea2df1eb.png">

whereas when i'm in "Send THIBCoin" and debug the value of `token` it displays:

<img width="471" alt="screen shot 2018-12-18 at 7 34 08 pm" src="https://user-images.githubusercontent.com/6226175/50168212-f01d2700-02fb-11e9-9a7b-a2a35052d759.png">

Reply:
i see what you mean, i think it makes sense the way you've done it then